src/main/java/de/bayern/gdi/gui/Controller.java

```diff
@@ -507,11 +523,28 @@ private void extractStoredQuery() {
 
     private void extractBoundingBox() {
         String bbox = "";
-        Envelope2D envelope;
+        Envelope2D envelope = null;
         if (this.dataBean.getServiceType().equals(ServiceType.Atom)) {
-            envelope = this.mapAtom.getBounds();
+            //in Atom the bboxes are given by the extend of every dataset
+            envelope = null;
```

Comment:
It's already null. Better use a switch ()this.dataBean.getServiceType() { ... } here.